src/tbb/governor.h

```diff
@@ -151,6 +158,68 @@ class governor {
         return false;
 #endif
     }
+
+#if __TBB_STATISTICS
+    enum class statistics : std::size_t {
+        exec_task = 0,
+        get_task,
+        task_res_not_null,
+        steal_task,
+        try_steal_task,
+        victim_pool_empty,
```

Comment:
```suggestion
        execute_task = 0,
        try_get_task,
        successfull_get_task,
        try_steal_task,
        successfull_steal_task,
        victim_pool_empty,
```
@alexey-katranov, @pavelkumbrasev what do you think about such renaming?